staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -168,6 +168,10 @@ func (a *mutatingDispatcher) Dispatch(ctx context.Context, attr admission.Attrib
 		if err != nil {
 			switch err := err.(type) {
 			case *webhookutil.ErrCallingWebhook:
+				if ctx.Err() == context.Canceled {
```

Comment:
calling ctx.Err() and getting an error after the webhook rejected does not mean the webhook rejected because of the cancellation, right? Shouldn't we be checking something inside the returned error to determine if it was cause by the cancellation?

Reply:
@divyasri537 this is a good point lets check if we can check this on the error itself